_posts/2019-01-20-why-we-run-our-blog.md

```diff
@@ -0,0 +1,179 @@
+in this case we'd be switching from a long-lived and mature set of dependencies to a single startup. This greatly
+increases your long-term risks.
+
+Medium is totally incentivized to get your post in front of as many people as possible, and that's awesome.
+However, as a business they've not found a way to be profitable, and have taken a lot of [VC cash][cb] which
+eventually needs to be paid back. (Artsy has too, so yeah, that's kinda hypocritical, but we're not aiming to
+disrupt the writing market we're augmenting the existing Art Industry)
+
+When newspapers like [Forbes][forbes] and [Bloomberg][bloomberg] are worried about their business model, then it
+doesn't look good for the longevity of your companies blog.
+
+## #branding
+
+{% include epic_img.html url="/images/hosting-our-own-blog/1.png" title="Screenshots" %}
+
+I consider our blog, and the rest of the site, to be the canonical representation of the Artsy Engineering team
+online. We've carefully grown an Artsy Engineering aesthetic around it s.
+
+In contrast, had we choose to host on Medium, we'd get a few templates and a highlight color. For example, check
+out: [AirBnB](https://medium.com/airbnb-engineering), [Harrys](https://medium.com/harrys-engineering),
+[Pintrest](https://medium.com/@Pinterest_Engineering), [Vimeo](https://medium.com/vimeo-engineering-blog) or
+[Lyft](https://eng.lyft.com)'s pages.
+
+Not memorable at all, because you can't really work with the design constraints to do anything creative other than
+a banner image.
+
+These constrains become worse on a post page, where you completely lose any sense of connection with the company,
+and the team the moment someone scrolls an inch until the footer. Your team's writing becomes just "a medium post"
+at that point. You've got limited options for attaching images, and no ability to use HTML/JS to showcase
+[problems][rn] [interactively][ar].
+
+These aren't blockers in any way, most of our posts don't do that - but the constraints mean you will never think
+to try and explain something outside of that sandbox.
+
+Then at the bottom of your post, readers are redirected to other posts from other teams. For example, I opened a
+post on Vimeo's announcement of [Psalm v3][psalm] (a cool looking PHP dev tool), there were three recommended
+posts: one was about missiles being fired in Syria, another was a beginners guide to PHP and then a third was how
+to set up Docker to work with a PHP framework. They're not that related, maybe they all have the keyword of PHP
+behind the scenes?
+
+{% include epic_img.html url="/images/hosting-our-own-blog/3.jpg" title="Artsy as a slideshow" %}
+
+Writing takes a lot of time, and provides so much value. It should be presented as [quality worthy of art][qwoa].
+By using to a generic platform for your writing, you're trading that simplicity for building your team's online
+presence.
+
+## Breaking the Sandbox
+
+We've grown to need to showcase quite a few different types of posts
```

Comment:
`different types of posts` could link to http://artsy.github.io/blog/2017/12/01/engineering-blog-post-templates/